assets/javascripts/discourse/widgets/discourse-reactions-reaction-button.js.es6

```diff
@@ -18,17 +18,9 @@ export default createWidget("discourse-reactions-reaction-button", {
     const currentUserReaction = this.attrs.post.current_user_reaction;
 
     if (!this.capabilities.touch) {
-      if (hasUsedMainReaction) {
-        this.callWidgetFunction("toggleLike");
-      } else if (currentUserReaction) {
-        this.callWidgetFunction("toggleReaction", {
-          reaction: currentUserReaction.id,
-          postId: this.attrs.post.id,
-          canUndo: currentUserReaction.can_undo
-        });
-      } else {
-        this.callWidgetFunction("toggleLike");
-      }
+      this.callWidgetFunction("toggleLike", {
```

Comment:
I don't get why we have to pass the reaction to toggleLike? toggleLike should be responsible for this logic.

Reply:
Oooh sorry, i forgot to change the method name😅